turbo.json

```diff
@@ -9,11 +9,10 @@
     // lint tasks
     "lint": {},
     // root lint task runs toml check and prettier
-    // TODO: run prettier in individual workspaces instead of globally.
     "//#lint": {
-      "dependsOn": ["//#check:toml", "//#check:prettier"]
+      "dependsOn": ["//#check:toml", "lint:prettier"]
     },
-    "//#check:prettier": {},
+    "lint:prettier": {},
```

Comment:
We should be smarter about setting `inputs` here. Make it just the places that `prettier` is going to search:

***

Update:

`prettier --support-info | jq "[.languages[].extensions[]] | unique"`

`pnpm prettier --support-info | jq "[.languages[].filenames | select( . != null )[]] | unique"`


Wow that's a lot of places, we should make this automatic.

```
[
  ".arcconfig",
  ".auto-changelog",
  ".babelrc",
  ".c8rc",
  ".clang-format",
  ".clang-tidy",
  ".devcontainer.json",
  ".eslintrc",
  ".eslintrc.json",
  ".gemrc",
  ".htmlhintrc",
  ".imgbotconfig",
  ".jscsrc",
  ".jshintrc",
  ".jslintrc",
  ".lintstagedrc",
  ".nycrc",
  ".prettierrc",
  ".stylelintrc",
  ".swcrc",
  ".tern-config",
  ".tern-project",
  ".watchmanconfig",
  "CITATION.cff",
  "Jakefile",
  "Pipfile.lock",
  "README",
  "api-extractor.json",
  "composer.json",
  "composer.lock",
  "contents.lr",
  "devcontainer.json",
  "glide.lock",
  "jsconfig.json",
  "language-configuration.json",
  "mcmod.info",
  "package-lock.json",
  "package.json",
  "tsconfig.json",
  "tslint.json"
]
```

```
[
  ".4DForm",
  ".4DProject",
  ".JSON-tmLanguage",
  "._js",
  ".avsc",
  ".bones",
  ".cjs",
  ".code-snippets",
  ".component.html",
  ".css",
  ".cts",
  ".es",
  ".es6",
  ".frag",
  ".geojson",
  ".gltf",
  ".gql",
  ".graphql",
  ".graphqls",
  ".gs",
  ".handlebars",
  ".har",
  ".hbs",
  ".hta",
  ".htm",
  ".html",
  ".html.hl",
  ".ice",
  ".importmap",
  ".inc",
  ".jake",
  ".javascript",
  ".js",
  ".js.flow",
  ".jsb",
  ".jscad",
  ".jsfl",
  ".jslib",
  ".jsm",
  ".json",
  ".json5",
  ".jsonc",
  ".jspre",
  ".jss",
  ".jsx",
  ".less",
  ".livemd",
  ".markdown",
  ".mcmeta",
  ".md",
  ".mdown",
  ".mdwn",
  ".mdx",
  ".mir",
  ".mjml",
  ".mjs",
  ".mkd",
  ".mkdn",
  ".mkdown",
  ".mts",
  ".njs",
  ".pac",
  ".pcss",
  ".postcss",
  ".reek",
  ".ronn",
  ".rviz",
  ".scd",
  ".scss",
  ".sjs",
  ".ssjs",
  ".sublime-build",
  ".sublime-commands",
  ".sublime-completions",
  ".sublime-keymap",
  ".sublime-macro",
  ".sublime-menu",
  ".sublime-mousemap",
  ".sublime-project",
  ".sublime-settings",
  ".sublime-syntax",
  ".sublime-theme",
  ".sublime-workspace",
  ".sublime_metrics",
  ".sublime_session",
  ".syntax",
  ".tfstate",
  ".tfstate.backup",
  ".topojson",
  ".ts",
  ".tsx",
  ".vue",
  ".webapp",
  ".webmanifest",
  ".workbook",
  ".wxs",
  ".wxss",
  ".xht",
  ".xhtml",
  ".xsjs",
  ".xsjslib",
  ".yaml",
  ".yaml-tmlanguage",
  ".yaml.sed",
  ".yml",
  ".yml.mysql",
  ".yy",
  ".yyp"
]
```